ci/run/run_bazel_test_with_sharding.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
 done
 
-SHARD=$(python ./ci/run/bazel_sharding/bazel_sharding.py --exclude_manual --index "${BUILDKITE_PARALLEL_JOB}" --count "${BUILDKITE_PARALLEL_JOB_COUNT}" --tag_filters="$test_tag_filters" "${targets[@]}")
+SHARD=$(python ./ci/ray_ci/bazel_sharding.py --exclude_manual --index "${BUILDKITE_PARALLEL_JOB}" --count "${BUILDKITE_PARALLEL_JOB_COUNT}" --tag_filters="$test_tag_filters" "${targets[@]}")
```

Comment:
should this be changed also to `bazel run`?

Reply:
probably we can wait until we replace all this with the ray_ci/runner script